### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Pub4Game** [Nukkit/Nukkit#76]
2015-11-27 17:32:08 [CRITICAL] Ошибка прорисовки мира "world": java.lang.IllegalArgumentException: Value -11.0 exceeds the range!
2015-11-27 17:32:16 [ALERT] java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(Unknown Source)
	at java.util.HashMap$ValueIterator.next(Unknown Source)
	at cn.nukkit.level.format.generic.BaseFullChunk.unload(BaseFullChunk.java:392)
	at cn.nukkit.level.format.mcregion.McRegion.unloadChunk(McRegion.java:264)
	at cn.nukkit.level.format.mcregion.McRegion.setChunk(McRegion.java:316)
	at cn.nukkit.level.Level.setChunk(Level.java:1822)
	at cn.nukkit.level.Level.generateChunkCallback(Level.java:1789)
	at cn.nukkit.level.generator.PopulationTask.onCompletion(PopulationTask.java:160)
	at cn.nukkit.scheduler.AsyncPool.collectTasks(AsyncPool.java:125)
	at cn.nukkit.scheduler.ServerScheduler.mainThreadHeartbeat(ServerScheduler.java:180)
	at cn.nukkit.Server.tick(Server.java:927)
	at cn.nukkit.Server.tickProcessor(Server.java:747)
	at cn.nukkit.Server.start(Server.java:728)
	at cn.nukkit.Server.<init>(Server.java:424)
	at cn.nukkit.Nukkit.main(Nukkit.java:65)

2015-11-27 17:32:16 [INFO] Stopping other threads

**Comment by caoli5288** [comment]
#73 Fixed.

**Comment by Pub4Game** [comment]
omg, I do not saw... I'm not be attentive... Tanks

**Comment by Pub4Game** [comment]
Don't want to open a new pull request... I have another error
![image](https://cloud.githubusercontent.com/assets/11244874/11443677/08bda272-9530-11e5-9f69-780f069394d6.png)

**Comment by Pub4Game** [comment]
Anyway...
![image](https://cloud.githubusercontent.com/assets/11244874/11443710/4c5e65c0-9530-11e5-9f70-66864dd24082.png)

**Comment by Angelic47** [comment]
It's hard to know how this exception caused.
Maybe this is a new bug we don't know, and need more information to debug. So could you please set debug level to 2 in nukkit.yml? This will let nukkit print stack trade to console, and it will more easier to find what went wrong.
Thanks

**Comment by Pub4Game** [comment]
2015-11-27 18:31:30 [INFO] Pub4Game[/192.168.1.172:19132] logged in with entity id 1 at (world, 128.0272, 4.0, 128.0941)
2015-11-27 18:31:32 [INFO] Pub4Game joined the game
2015-11-27 18:31:33 [INFO] Pub4Game fell from a high place
2015-11-27 18:31:33 [ALERT] java.lang.IllegalArgumentException: Value -11.0 exceeds the range!
```
	at cn.nukkit.entity.Attribute.setValue(Attribute.java:121)
	at cn.nukkit.Player.setHealth(Player.java:2907)
	at cn.nukkit.entity.Entity.attack(Entity.java:591)
	at cn.nukkit.entity.Living.attack(Living.java:96)
	at cn.nukkit.Player.attack(Player.java:2968)
	at cn.nukkit.entity.Entity.fall(Entity.java:952)
	at cn.nukkit.entity.Entity.updateFallState(Entity.java:935)
	at cn.nukkit.entity.Entity.move(Entity.java:1161)
	at cn.nukkit.Player.processMovement(Player.java:1192)
	at cn.nukkit.Player.onUpdate(Player.java:1373)
	at cn.nukkit.level.Level.doTick(Level.java:542)
	at cn.nukkit.Server.checkTickUpdates(Server.java:867)
	at cn.nukkit.Server.tick(Server.java:929)
	at cn.nukkit.Server.tickProcessor(Server.java:747)
	at cn.nukkit.Server.start(Server.java:728)
	at cn.nukkit.Server.<init>(Server.java:424)
	at cn.nukkit.Nukkit.main(Nukkit.java:65)
```
2015-11-27 18:31:33 [CRITICAL] Could not tick level "world": java.lang.IllegalArgumentException: Value -11.0 exceeds the range!

**Comment by Angelic47** [comment]
Thank you! We will fix it soon :D

**Comment by Angelic47** [comment]
Maybe fixed now :D #78

**Comment by asyven** [comment]
@Snake1999 i keep crashing by this, sometimes
